client/checker_client/cli.py

```diff
@@ -0,0 +1,229 @@
+import click
+
+from checker_client import checker as checker_lib
+import pytezos
+from pathlib import Path
+from dataclasses import dataclass
+from marshmallow import Schema, fields
+from marshmallow.decorators import post_load
+import portpicker
+import json
+
+CONFIG_FILE = Path.home().joinpath(Path(".checker"))
+
+
+@dataclass
+class Config:
+    path: str
+    sandbox_port: int = 18731
+    sandbox_container: str = "checker-sandbox-node"
+    tezos_address: str = "http://127.0.0.1"
+    tezos_port: int = 18731
+    tezos_key: str = "edsk3RFfvaFaxbHx8BMtEW1rKQcPtDML3LXjNqMNLCzC3wLC1bWbAt"  # bob's private key from "edobox info"
+    ctez_address: str = ""
+    oracle_address: str = ""
+    checker_address: str = ""
+
+    @staticmethod
+    def load(file: Path):
+        with file.open("r") as f:
+            return ConfigSchema().loads(f.read())
+
+    def dump(self):
+        with open(self.path, "w") as f:
+            f.write(ConfigSchema().dumps(self))
```

Comment:
I had concerns first time I read this config-mutating stuff, but once I actually used the CLI from the command line I can see that it is super useful. I might steal this pattern in future :).

Reply:
Thanks! This was also my first time trying this. I also had some doubts, but it really is convenient. I think `gcloud` does something similar which is kind of where I got the idea from.